src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+  justify-content: space-between;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-position: center;
+    background-repeat: no-repeat;
```

Comment:
it can be minimized in just `background` shorthand property
https://www.w3schools.com/css/css_background_shorthand.asp